### PR TITLE
sound stuff

### PR DESCRIPTION
**Pull request by StickWolf**
CAN YOU CHECK THISSS!!!!??

### CODE REVIEW

**Review by XanderSuFromQBIX — DISMISSED** [review]

File: SoundUtils1/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+        public void HealBeeps()
+        {
+            Console.Beep(800, 100);
+            Console.Beep(700, 100);
+            Console.Beep(600, 100);
+            Console.Beep(700, 100);
+        }
+
+        public void AcientPianoBeeps()
```
Comment (Materia-xx) [inline SoundUtils1/Class1.cs]: Nit: Ancient

**Comment by XanderSuFromQBIX** [comment]
### _**Great! You Did It!**_

File: SoundUtils1/Class1.cs (hunk 1 above)
Comment (StickWolf) [reply SoundUtils1/Class1.cs]: it is acient

File: SoundUtils1/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SoundUtils1
```
Comment (Materia-xx) [inline SoundUtils1/Class1.cs]: Please don't name your library or programs ending with a 1

File: SoundUtils1/Class1.cs (hunk 1 above)
Comment (StickWolf) [reply SoundUtils1/Class1.cs]: ### Acient

**Review by Materia-xx — CHANGES_REQUESTED** [review]
Please fix the comments and submit another iteration

File: SoundUtils1/Class1.cs (hunk 1 above)
Comment (XanderSuFromQBIX) [reply SoundUtils1/Class1.cs]: ### 

> _**Really? It is??**_

Comment (XanderSuFromQBIX) [reply SoundUtils1/Class1.cs]: This is a test
`Console.WriteLine("Test");`

**Review by Materia-xx — APPROVED** [review]

**Review by XanderSuFromQBIX — APPROVED** [review]

**Pull request merged**